r/R/csv.R

```diff
@@ -143,6 +143,17 @@
 #' read_csv_arrow(tf, schema = schema(x = int32(), y = utf8()), skip = 1)
 #' read_csv_arrow(tf, col_types = schema(y = utf8()))
 #' read_csv_arrow(tf, col_types = "ic", col_names = c("x", "y"), skip = 1)
+#'
+#' # Note that if a timestamp column contains time zones, type inference won't work,
+#' # whether automatic or via the string "T" `col_types` specification.
```

Comment:
Sorry, I realize this was not the proper wording. It can be read as an automatic guess.
I have opened a PR to correct this, please merge if you would like. #14413